src/Shared/TaskHostConfiguration.cs

```diff
@@ -44,6 +44,11 @@ internal class TaskHostConfiguration : INodePacket
         private CultureInfo _uiCulture = CultureInfo.CurrentUICulture;
 
 #if FEATURE_APPDOMAIN
+        /// <summary>
+        /// The app domain configuration bytes sent via RPC.
+        /// </summary>
+        private byte[] _appDomainConfigBytes;
```

Comment:
Looks unused outside of `Translate`. Can it be a local variable there?

Reply:
Sure, it was remnant of the original implementation.